sky/provision/gcp/instance_utils.py

```diff
@@ -75,10 +79,30 @@ def filter(
     def delete_firewall_rule(
         cls,
         project_id: str,
-        firewall_rule_name: str,
+        cluster_name_on_cloud: str,
```

Comment:
why is the inherited class has the old argument?

Reply:
Sry it was some remnant. Thanks for pointing that out!